lib/main_common.pm

```diff
@@ -471,7 +471,10 @@ sub load_zdup_tests {
     # Restrict version switch to sle until opensuse adopts it
     loadtest "migration/version_switch_upgrade_target" if is_sle and get_var("UPGRADE_TARGET_VERSION");
     loadtest 'boot/boot_to_desktop';
-    loadtest 'console/check_upgraded_service' if !is_desktop;
+    if (!is_desktop) {
+        loadtest 'console/check_upgraded_service';
+        loadtest 'console/check_package_version';
```

Comment:
add if(is_sle(12sp5) since this module only check version number for 12sp5 project.